clients/bigquery/cast.go

```diff
@@ -59,11 +60,23 @@ func castColVal(colVal any, colKind columns.Column, additionalDateFmts []string)
 			return fmt.Sprintf(`{"key":"%s"}`, constants.ToastUnavailableValuePlaceholder), nil
 		}
 
-		colValString, isOk := colVal.(string)
```

Comment:
Would it make sense to move the check above for the `constants.ToastUnavailableValuePlaceholder` to the `if` statement below? Otherwise we're potentially calling `fmt.Sprint()` on a map.